test/validators.js

```diff
@@ -4190,10 +4191,18 @@ describe('Validators', () => {
           '235407195106112745',
           '210203197503102721',
           '520323197806058856',
-          '235477190110989',
+          '110101491001001',
```

Comment:
Can add a note for this churn?

Reply:
Both values are correct, i think the old version is an improvements made by @ezkemboi  to cover 15 characters ID cards that got overwritten when i merged my own existing changes. I Can undo it if you want to